packages/babel-preset-fbjs/plugins/inline-requires.js

```diff
@@ -90,6 +97,12 @@ module.exports = babel => ({
               for (const referencePath of binding.referencePaths) {
                 excludeMemberAssignment(moduleName, referencePath, state);
                 try {
+                  const requireBinding = referencePath.scope.getBinding(
+                    requireName
+                  );
+                  if (requireBinding != null) {
+                    requireBinding.scope.rename(requireName);
+                  }
```

Comment:
The check for an existing `requireBinding` shouldn't be necessary. `referencePath.scope.rename` is a no-op if the existing name has no references.

```suggestion
                  referencePath.scope.rename(requireName);
```